docs/source/markdown/podman-container-restore.1.md

```diff
@@ -4,120 +4,131 @@
+The default is **false**.\
+*IMPORTANT: This OPTION is only available in combination with **--import, -i**.*
 
 #### **--ignore-static-ip**
 
-If the container was started with **--ip** the restored container also tries to use that
+If the *container* was started with **--ip** the restored *container* also tries to use that
 IP address and restore fails if that IP address is already in use. This can happen, if
-a container is restored multiple times from an exported checkpoint with **--name, -n**.
+a *container* is restored multiple times from an exported checkpoint with **--name, -n**.
 
 Using **--ignore-static-ip** tells Podman to ignore the IP address if it was configured
-with **--ip** during container creation.
+with **--ip** during *container* creation.
+
+The default is **false**.
 
 #### **--ignore-static-mac**
 
-If the container was started with **--mac-address** the restored container also
+If the *container* was started with **--mac-address** the restored *container* also
 tries to use that MAC address and restore fails if that MAC address is already
-in use. This can happen, if a container is restored multiple times from an
+in use. This can happen, if a *container* is restored multiple times from an
 exported checkpoint with **--name, -n**.
 
 Using **--ignore-static-mac** tells Podman to ignore the MAC address if it was
-configured with **--mac-address** during container creation.
+configured with **--mac-address** during *container* creation.
+
+The default is **false**.
 
 #### **--ignore-volumes**
 
 This option must be used in combination with the **--import, -i** option.
-When restoring containers from a checkpoint tar.gz file with this option,
-the content of associated volumes will not be restored.
+When restoring *containers* from a checkpoint tar.gz file with this option,
+the content of associated volumes will not be restored.\
+The default is **false**.
 
-#### **--publish**, **-p**
+#### **--import**, **-i**=*file*
 
-Replaces the ports that the container publishes, as configured during the
-initial container start, with a new set of port forwarding rules.
+Import a checkpoint tar.gz file, which was exported by Podman. This can be used
+to import a checkpointed *container* from another host.\
+*IMPORTANT: This OPTION does not need a container name or ID as input argument.*
 
-```
-# podman run --rm -p 2345:80 -d webserver
-# podman container checkpoint -l --export=dump.tar
-# podman container restore -p 5432:8080 --import=dump.tar
-```
+#### **--import-previous**=*file*
 
-For more details please see **podman run --publish**.
+Import a pre-checkpoint tar.gz file which was exported by Podman. This option
+must be used with **-i** or **--import**. It only works on `runc 1.0-rc3` or `higher`.
```

Comment:
@mheon @rhatdan is the runc requirement true?  Doesn't this work on crun too?

Reply:
An answer would be great. Everything else is ready to go.